show/main.py

```diff
@@ -38,6 +40,45 @@ def read_config(self, filename):
             pass
 
 
+class Alias(object):
+    """Object to Alias conversion"""
+
+    def __init__(self):
+        self.vendor_name_length = 0
+        cmd = 'sonic-cfggen -d --var-json "PORT"'
+        p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+        self.port_dict = json.loads(p.stdout.read())
+
+        for port_name in self.port_dict.keys():
+            if self.vendor_name_length < len(
+                    self.port_dict[port_name]['alias']):
+               self.vendor_name_length = len(
+                    self.port_dict[port_name]['alias'])
+
+    def interface_name_to_alias(self, interface_name):
+        """Return alias interface name if default
+           name is given as argument
+        """
+        if interface_name is not None:
+            for port_name in self.port_dict.keys():
+                if interface_name == port_name:
+                    return self.port_dict[port_name]['alias']
+        click.echo("Invalid interface {}".format(interface_name))
+
```

Comment:
Remove blank line 67. Consider adding blank line between 65 and 66.

Reply:
👍